webapp/src/blocks.tsx

```diff
@@ -551,6 +551,9 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     insertBreakpoint() {
         if (!this.editor) return;
         const b = this.editor.newBlock(pxtc.TS_DEBUGGER_TYPE);
+        // move block roughly to the center of the screen
+        const m = this.editor.getMetrics();
+        b.moveBy(m.viewWidth / 2, m.viewHeight / 3);
```

Comment:
You can use b.getHeightWidth() to get the size of the block and incorporate that into the calculation if you like